tests/ui/decimal_bitwise_operands.stderr

```diff
@@ -0,0 +1,164 @@
+error: using decimal literal for bitwise operation
+  --> tests/ui/decimal_bitwise_operands.rs:15:9
+   |
+LL |     x & 99;
+   |         ^^
+   |
+   = help: use binary (0b1100011), hex (0x63), or octal (0o143) notation for better readability
+   = note: `-D clippy::decimal-bitwise-operands` implied by `-D warnings`
+   = help: to override `-D warnings` add `#[allow(clippy::decimal_bitwise_operands)]`
+
+error: using decimal literal for bitwise operation
+  --> tests/ui/decimal_bitwise_operands.rs:16:23
+   |
+LL |     x | (/* comment */99);
+   |                       ^^
+   |
+   = help: use binary (0b1100011), hex (0x63), or octal (0o143) notation for better readability
```

Comment:
Would it make sense to group 4 or at least 8 binary digits by underscore? Makes it easier to read and avoids having another lint triggering on applying this one.